rudp/connection.py

```diff
@@ -34,8 +34,17 @@ def on_reset(data):
     def send(self, data):
         self._sender.send(data)
 
+        # TODO: we have no way of knowing for now what the --log-level specified passed was.
+        if self.log.level >= 0:
```

Comment:
ideally this would be something like:

```
if self.log.level >= 20
```

temporary code while we clean and refactor before the release.